common/models/templates/package.json

```diff
@@ -19,7 +19,10 @@
   "main": "./build/obj/index.js",
   "types": "./build/obj/index.d.ts",
   "exports": {
-    ".": "./build/obj/index.js",
+    ".": {
+      "es6-bundling": "./src/index.ts",
```

Comment:
Many thanks to https://github.com/evanw/esbuild/issues/1250#issuecomment-1463826174 for the approach used in this PR.

Reply:
Leveraging this feature during TypeScript compilation requires TS 5.0: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-0.html#customconditions

We're currently using 4.9.5, so we can't leverage that during ES5 builds... not that we've needed to, anyway.